src/entity/active_model.rs

```diff
@@ -36,6 +38,7 @@ pub use ActiveValue::NotSet;
 /// );
 /// ```
 #[derive(Clone, Debug)]
+#[cfg_attr(feature = "with-json", derive(Serialize, Deserialize))]
```

Comment:
We can use sea_query's `value_to_json` to handle the `Serialize` part, but in general ActiveValue does not seem to lend itself for `Deserialize`